main.py

```diff
@@ -27,6 +27,13 @@ def UpdateQueue():
         api.request('statuses/retweet/:' + str(post))
         post_list.pop(0)
 
+def CheckForFollowRequest(item):
+    text = item['text']
+    if "Follow" in text:
```

Comment:
`if "follow" in text.lower():` is safer because it also catches lower case text.
Actually using a regex and word boundaries would be even better.